include/cudaq/Optimizer/Dialect/Quake/QuakeOps.td

```diff
@@ -833,7 +833,7 @@ def ExpPauliOp : QuakeOp<"exp_pauli"> {
   let arguments = (ins
     AnyFloat:$parameter,
     VeqType:$qubits,
-    cc_PointerType:$pauli
+    AnyTypeOf<[cc_PointerType, PauliWordType]>:$pauli
```

Comment:
I'm a little confused by this.

In the context of quake, where does a value of `PauliWordType` come from? Is it correct that this is the only operation that using this type?

Below it looks like this new type is just an alias for a `i8*`. Is that true? Are we going to allow other compositions of values to have this type eventually?

Reply:
Should we be phasing out the `i8*` entirely? Maybe add a note to that intention?